### ISSUE TITLE
Bug - don´t create account

### ISSUE THREAD

**Issue by nelzijunior** [ethereum/mist#2212]
Hello guys!
I downloaded the Ethereum Wallet 32bits in a zip file.
Extracted on desktop and open the file Ethereum Wallet.exe (application).
The app openned and I click "add account".
Ask a password... I enter.. re enter.. press "OK".. 
And nothing happened!
How I sign in my account? Have no "Log in" or something else like that.
Anyone can help me?

**Comment by sathishvj** [comment]
No, there is no 'Log In' like certain other apps.  But you need a password to decrypt the private key that is in the keystore file that is used to verify that you own the account.  In the case of Mist it will ask you for the password when you want to make transactions.  Also, your password for different Ethereum addresses/accounts could be different.
